src/client/fontengine.h

```diff
@@ -112,15 +112,6 @@ class FontEngine
 	/** current font engine mode */
 	FontMode m_currentMode = FM_Standard;
 
-	/** font mode of last request */
```

Comment:
Replaced by replacing several `m_font_cache[mode]` lookups in `getFont()`. Roughly the same performance, I guess.